base_location_nuts/wizard/nuts_import.py

```diff
@@ -193,10 +193,9 @@ def run_import(self):
         xmlcontent = self._download_nuts()
         dom = etree.fromstring(xmlcontent)
         for node in dom.iter('Item'):
-                logger.debug(
-                    'Reading level=%s, id=%s',
-                    node.get('idLevel', 'N/A'),
-                    node.get('id', 'N/A'))
+                logger.debug('Reading level=%s, id=%s',
```

Comment:
The indentation problem I was referring to is that these are 8 spaces instead of the usual 4. (I mean from the line 195 to 196).